.storybook/deprecated/cyclebutton/cyclebutton.stories.js

```diff
@@ -42,5 +45,39 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
+export const Default = (({
+	rootClass = "spectrum-CycleButton",
+	customClasses = [],
+	size = "m",
+	initialIcon = "Play",
+	selectedIcon = "Pause",
+	isDisabled = false,
+	onclick,
+	...globals
+}) => {
+	const [{ selectedIcon: icon }, updateArgs] = useArgs();
+
+	return html`
+		<style>
+			:root, .spectrum--medium { --spectrum-global-dimension-size-85: 7px; }
```

Comment:
I manually pulled in the vars tokens so we didn't have to load the asset for these few deprecated components.